### DISCUSSION TITLE
Add icons or buttons in a detailslist

### DISCUSSION THREAD

**Discussion by LeoHammett** [microsoft-search/pnp-modern-search#3734] [General]
Hello there!

I´m a beginner on these web parts and I´m not sure how to add a new icon/button bellow of each item in the details-list 

This is my item list:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/83602019/0b49dba4-63f8-4c6e-a781-288cabe6f73d)
 
And this is how I would like it to look:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/83602019/5b695b15-7749-455f-8ee7-716716b393d0)

Does anyone have a hint of what can I do, or how can I start?

Thanks in advance!

**Comment by kasperbolarsen** [comment]
Hi, you might find inspiration both in the documentation for handlebars https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/ and in the samples repository, like this one https://github.com/microsoft-search/pnp-modern-search-layouts/blob/main/Results/Microsoft%20Search%20Style%20Documents/Microsoft%20Search%20Documents%20layouts.md

**Comment by wobba** [comment]
You can use `<pnp-icon>` in your templates. https://microsoft-search.github.io/pnp-modern-search/extensibility/web_components_list/#pnp-icon
